### DISCUSSION TITLE
Quorum for DCDAO Governance

### DISCUSSION THREAD

**Discussion by SasoLithops** [ringdao-community-guild/discussion#69] [Proposals Temperature Check]
As DCDAO Constitution created by Cindy passes (link below): 

https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xd750140eccee3434f65feb156c985f6425ed884d56aa2a295c189d5e76fe9dc5 

I propose raising quorum to achieve decentralization. 

Additionally, DCDAO members who have been carefully chosen and approved by the DCDAO Steering Committee should participate in governance and discussions. 

The quorum should not be so easily reached (with many new members) because it should be based on a wider range of opinions and consensus.  

Thus, I ask you to participate in the poll voting and share your opinion on this topic. 

Here are the options I will poll next to this proposal(feel free to let me know in comments below if on official proposal I need to add some other options):

1. keep 4 votes
2. 50%
3. 60%
4. 67% (2/3 of all members)

There will be a two-week poll voting period for this temperature check proposal.

P.S. Special thank you to David(Maduksjay) for helping me with grammar and other mistakes I did.

**Comment by casinoyoda** [comment]
I believe that augmenting the Quorum is acceptable, but it necessitates the establishment of a dynamic adjustment mechanism. This mechanism should incorporate factors such as the total number of members and the number of actively participating voters.

The term "quorum" refers to the requisite minimum number of group members that must be present to validate the proceedings of any proposal. If this threshold is excessively high, it could present challenges in gathering sufficient votes to propel the proposal forward.

Currently, our group comprises 15 members. However, only about 5 or 6 are consistently active voters, as evidenced by Snapshot. Thus, I propose that for the time being, setting the quorum to 7 or 8 would be adequate.

**Reply by SasoLithops** [reply]
Is this dynamic adjustment mechanism hard to code? Could we then just do it manually when new people join?

All valid points. And although I voted for 2/3(66%) I will absolutley accept what DCDAO will agree upon. 

I think it is 14(with Cindy leaving).  I know some proposals will be hard to get to quorum, however engaging more people into being active participants(and we know all members of DCDAO and 9 of us are from DC(5 of team and 4 ambassadors). I also feel by governance you learn about progress of DCDAO and get more engaged into discussions.

I will add one of my experiences here. In ION DAO where I participate we keep 30% quorum. However there are many holders that made it in crypto and left. So dev team have started writting to every wallet that holds more ION to vote. I think if all that still activly hold voted it could barely reach 40%. But we still keep high quorum to get community engaged. So I would rather see we are taging people(I can do it) and asking them to vote, then allow many important proposals to pass with only quorum votes.

**Comment by Laki389** [comment]
I would keep the quorum at 4 votes.

IMHO, important proposals, or controversial proposals, naturally attract more voters, lobbying to vote, and copy-paste proposals that are repeated and this quorum of 4 voters is now difficult to reach.

Here for example, this proposal has only 3 votes in total, the voting is over and if it goes according to the rules and regulations, the voting would be invalid.
https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xcbe5bce58db3cf1ba6de432603447631858545a2dc933130af3ffc02ebdc5c01

**Reply by SasoLithops** [reply]
Again we are on separate hills. And to emphasize that is ABSOLUTELY fine. 

However to debate further and express why I find this bad, keeping such low quorum will keep away people for participating in governance. We don't have validators/collators that vote for us. So we need to be active. My point is that people need to be active in governance and debates to push things forward. With Opengov we will have discussions before proposal goes live and need that active participation. Not one or two talking to themselves. 

And the example you gave is exactlly how it is now. No one knows, no one cares. Just make it pass....

**Reply by Laki389** [reply]
Putting a quorum of 8 or 9 DCDAO members, IMHO, is not a magic wand that will suddenly increase the number of voters.
In my opinion, it will only increase the number of invalid proposals, i.e. it will probably not be possible to gather the quorum for almost any proposal (except for some very very very important voting).

Looking at all the proposals so far, only one vote had 8 votes, most are 4 or 5.
Even, there is one with 3. (if we are consistent with what we adopted as DCDAO rules, even this proposal would be, and is, invalid.

**Reply by SasoLithops** [reply]
My experiences speak othervise but your opinion much appreciated. Let's see what others have to say.

![Screenshot_2024-01-30-07-05-03-114_org.telegram.messenger.jpg](https://github.com/dcdao/discussion/assets/101176902/4e84e53f-3193-45e5-bc9d-1ad0f20c5b34)

**Reply by Laki389** [reply]
> My experiences speak othervise but your opinion much appreciated. Let's see what others have to say.
> 


I am not very familiar with ION governance, do they vote with their tokens (if the project has tokens) or with NFT`s? 
(like DCDAO, 1 person = 1 vote, DCDAO member may or may not have Ring/Kton at all)

**Reply by SasoLithops** [reply]
With tokens. So much harder to tag everyone to vote😉
But persistance and education with amazing community engagement did the trick. ION DAO launched IBCX(index of Cosmos eco tokens) and proposal passing quorum in few day for second revamping. I belive it is great example for how different opinions that were disscussed lead to incredible result.

YoudleDAO is next one to look up to. Great joined engagement from community and Invarch team and building strong active community.We will be choosing 30 new members by voting.

Hope others share their experiences on governance here too.

Also what Phil posted is interested what Hydra does with governance participants. Being different than others is the key:

You can stake their token and it's used for fees as well as governance. You also get boosters to your staking for participating in governance.

**Reply by Laki389** [reply]
If they vote with tokens, I'm afraid the comparison isn't exactly the most appropriate.

Then it is even more difficult to achieve some predetermined quorum.
Some projects achieve a high percentage of votes, by choosing the validator during staking, they choose a delegate to vote on their behalf. (the validator votes on their behalf)
This, in my opinion, gives great influence to a small group of people - validators, who do not necessarily (always) vote in the interest of those who own those tokens and whose interests they (should) represent.

**Reply by SasoLithops** [reply]
That is the hack. We set it high intentionaly so small groups can't do nothing. And it is/was very hard to reach quorum. But we insisted on this so that people start to care and participate in voting if we want to execute new things. We all in this invested together, we are a DAO not council.
JFYI Ion has no validators. It has no staking rewards. We only stake ION to participate in governance. And it works. Was surprising to me too. 

My only concern in not keeping low quorum in DCDAO is to keep people engaged, active. 

Hope you understand my point now.
